joern-cli/frontends/swiftsrc2cpg/src/test/scala/io/joern/swiftsrc2cpg/passes/ast/AsyncSyntaxTests.scala

```diff
@@ -3,28 +3,48 @@
 package io.joern.swiftsrc2cpg.passes.ast
 
 import io.joern.swiftsrc2cpg.testfixtures.AstSwiftSrc2CpgSuite
-
-import io.joern.swiftsrc2cpg.testfixtures.AstSwiftSrc2CpgSuite
+import io.shiftleft.codepropertygraph.generated.*
+import io.shiftleft.codepropertygraph.generated.nodes.*
+import io.shiftleft.semanticcpg.language.*
+import io.shiftleft.semanticcpg.language.types.structure.NamespaceTraversal
 
 class AsyncSyntaxTests extends AstSwiftSrc2CpgSuite {
 
   "AsyncSyntaxTests" should {
 
-    "testAsyncSyntax1" ignore {
+    "testAsyncSyntax1" in {
       val cpg = code("""
         |func asyncGlobal1() async { }
         |func asyncGlobal2() async throws { }
         |""".stripMargin)
-      ???
+      val List(asyncGlobal1, asyncGlobal2) = cpg.method.internal.nameNot(NamespaceTraversal.globalNamespaceName).l
+      asyncGlobal1.name shouldBe "asyncGlobal1"
+      asyncGlobal1.fullName shouldBe "Test0.swift:<global>:asyncGlobal1"
+      asyncGlobal2.name shouldBe "asyncGlobal2"
+      asyncGlobal2.fullName shouldBe "Test0.swift:<global>:asyncGlobal2"
     }
 
-    "testAsyncSyntax2" ignore {
+    "testAsyncSyntax2" in {
       val cpg = code("""
         |typealias AsyncFunc1 = () async -> ()
         |typealias AsyncFunc2 = () async throws -> ()
         |typealias AsyncFunc3 = (_ a: Bool, _ b: Bool) async throws -> ()
         |""".stripMargin)
-      ???
+      val List(f1, f2, f3) = cpg.typeDecl.where(_.aliasTypeFullName).l
+      f1.name shouldBe "AsyncFunc1"
+      f1.fullName shouldBe "Test0.swift:<global>:AsyncFunc1"
+      f1.aliasTypeFullName shouldBe Option("Test0.swift:<global>:_anon_cdecl_0")
```

Comment:
seems this isn't new in this PR, but it would be good if these generated names had some invalid characters in them, to guarantee that they can't clash with names from actual code. (Like with `<operator>` and `<lambda>`.)

Reply:
Will do.